lms/djangoapps/courseware/models.py

```diff
@@ -260,3 +260,20 @@ class Meta:
 
     def __unicode__(self):
         return "[OCGLog] %s: %s" % (self.course_id.to_deprecated_string(), self.created)  # pylint: disable=no-member
+
+
+class StudentFieldOverride(models.Model):
+    """
+    Holds the value of a specific field overriden for a student.  This is used
+    by the code in the `courseware.student_field_overrides` module to provide
+    overrides of xblock fields on a per user basis.
+    """
+    course_id = CourseKeyField(max_length=255, db_index=True)
+    location = LocationKeyField(max_length=255, db_index=True)
+    student = models.ForeignKey(User, db_index=True)
+
+    class Meta:   # pylint: disable=missing-docstring
+        unique_together = (('course_id', 'location', 'student'),)
+
+    field = models.CharField(max_length=255)
+    value = models.TextField(default='null')
```

Comment:
I think my inclination for this model would be to make it a bit more specific. In particular, I think I'd have a table for `POCFieldOverrides` with columns for each field that needs to be overridden. That will let you use the appropriate database types for the field types, and will make it easier to select everything you need for a particular student/course/location.